app/controllers/dashboards_controller.rb

```diff
@@ -3,7 +3,6 @@ class DashboardsController < BaseController
   before_action :set_policy_with_context, except: [:refresh_dashboard]
   before_action :set_dashboard
   before_action :set_defence_requests, only: [:show, :closed]
-  before_action :set_dashboard_count, only: [:show, :closed]
```

Comment:
I'm thinking that maybe dashboard could be `resources` instead of `resource` and that we could have different types like that. It would be more RESTful and I think maybe cleaner. It could help also with the pre-loading of different lists of defence requests, because it could be a scope with the same name (or something like that).

But that's just an idea, I'm not insisting.